src/protocol.rs

```diff
@@ -481,7 +484,24 @@ mod tests {
             Err(panic) => {
                 if let Some(s) = panic.downcast_ref::<String>() {
                     println!("Panic occurred: {s}");
-                    assert_eq!(s, "Participant Config Size Error");
+                    assert_eq!(s, "Received a message with too few parties (Participant Config should have at least 2 parties)");
+                }
+            }
+        }
+
+        let result = std::panic::catch_unwind(|| {
+            let mut rng = init_testing();
+            let _p = ParticipantConfig::random_quorum(1, &mut rng);
+        });
+
+        match result {
+            Ok(_) => {
+                println!("No panic occurred.");
+            }
+            Err(panic) => {
```

Comment:
todo: This looks like the same kind of issue from [your last PR](https://github.com/boltlabs-inc/tss-ecdsa/pull/301#discussion_r1170490024): this test won't fail if the `random_quorum` function doesn't panic, but part of the test is to make sure that it _does_ panic on these bad inputs. I would suggest the same approach as before: have two asserts, one checking that there is an error, and the other checking that it's the specific string.

In this case, the second one is less important because this is an internal interface, so we don't require a specific, user-friendly error (although you've produced one, which is nice). But you can check it anyway.